File: src/main/java/com/clinica/atendimento/repository/ResponsavelPacienteRepository.java
package com.clinica.atendimento.repository;

import com.clinica.atendimento.model.ResponsavelPaciente;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ResponsavelPacienteRepository extends JpaRepository<ResponsavelPaciente, Long> {
}

File: src/main/kotlin/com/clinica/atendimento/repository/ResponsavelPacienteRepository.kt
package com.clinica.atendimento.repository

import com.clinica.atendimento.model.ResponsavelPaciente
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ResponsavelPacienteRepository : JpaRepository<ResponsavelPaciente, Long>